internal_deps.bzl

```diff
@@ -7,19 +7,9 @@ statement from these, that's a bug in our distribution.
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 load("@bazel_tools//tools/build_defs/repo:utils.bzl", "maybe")
 
-
 def rules_container_internal_deps():
     "Fetch deps needed for local development"
-    maybe(
-        http_archive,
-        name = "rules_pkg",
-        urls = [
-            "https://mirror.bazel.build/github.com/bazelbuild/rules_pkg/releases/download/0.5.1/rules_pkg-0.5.1.tar.gz",
-            "https://github.com/bazelbuild/rules_pkg/releases/download/0.5.1/rules_pkg-0.5.1.tar.gz",
-        ],
-        sha256 = "a89e203d3cf264e564fcb96b6e06dd70bc0557356eb48400ce4b5d97c2c3720d",
-    )
-    
```

Comment:
Atm, we use rules_pkg to build tar for layers. Perhaps this should go into `rules_container_deps`?

Reply:
yes, I see, it's a runtime dependency so yes I'll move it